services/minespace-web/src/components/pages/Project/DocumentsTab.js

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import { withRouter } from "react-router-dom";
+import { Row, Col, Tabs, Typography } from "antd";
+import PropTypes from "prop-types";
+import customPropTypes from "@/customPropTypes";
+import { formatUrlToUpperCaseString } from "@common/utils/helpers";
+import { getProject } from "@common/selectors/projectSelectors";
+import { fetchProjectById } from "@common/actionCreators/projectActionCreator";
+import DocumentsPage from "./DocumentsPage";
+
+const propTypes = {
+  match: PropTypes.shape({
+    params: PropTypes.shape({
+      projectGuid: PropTypes.string,
+    }),
+  }).isRequired,
+  history: PropTypes.shape({
+    push: PropTypes.func,
+  }).isRequired,
+  project: customPropTypes.project.isRequired,
+  fetchProjectById: PropTypes.func.isRequired,
+};
+
+const tabs = ["project-description", "information-requirements-table", "major-mine-application"];
+
+export class DocumentsTab extends Component {
+  state = {
+    activeTab: tabs[0],
+  };
+
+  allDocuments = [];
+
+  componentDidMount() {
+    this.handleFetchData();
+    this.allDocuments.push(this.props.project?.project_summary?.documents);
+    this.allDocuments.push(this.props.project?.information_requirements_table?.documents);
+    this.allDocuments.push(this.props.project?.major_mine_application?.documents);
+  }
+
+  handleFetchData = () => {
+    const { projectGuid } = this.props.match?.params;
+    return this.props.fetchProjectById(projectGuid);
+  };
+
+  handleTabChange = (activeTab) => {
+    const { projectGuid } = this.props.match.params;
+    const url = `/projects/${projectGuid}/documents/#${activeTab}`;
+    this.props.history.push({ pathname: url });
+
+    const a = document.createElement("a");
```

Comment:
Why are we creating an anchor and clicking it here? My understanding was that the history.push() should suffice.